seqr/views/apis/anvil_workspace_api.py

```diff
@@ -240,3 +268,89 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+
+def generate_AnVIL_variables(project, data_path, sample_type):
+    pipeline_dag = {
```

Comment:
in general, helper functions should either encapsulate functionality that is reused in many places, or functionality that is complicated. One-liners that are only used once are better as inline code than helper functions